src/vuestic-theme/vuestic-components/va-list/VaList.demo.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <VbDemo>
+    <div class="va-list-demo">
+      <VaList :items="items" :dividers="true" @item-click="onClick"/>
```

Comment:
Exact design is a requirement.

![image](https://user-images.githubusercontent.com/5394573/56367495-041b5b80-61fe-11e9-84d9-5b88fbefaa88.png)